infra/eligibility_screener/template/ecs.tf

```diff
@@ -28,6 +40,89 @@ resource "aws_security_group" "allow-screener-traffic" {
   }
 }
 
+resource "aws_security_group" "allow-lb-traffic" {
+  name        = "screener_load_balancer_sg"
+  description = "Allows load balancers to communicate with tasks"
+  vpc_id      = module.constants.vpc_id
+
+  ingress {
+    description      = "HTTP traffic from VPC"
+    from_port        = 80
+    to_port          = 80
+    protocol         = "tcp"
+    cidr_blocks      = ["0.0.0.0/0"]
+    ipv6_cidr_blocks = ["::/0"]
+  }
+
+  ingress {
+    description = "allow lb traffic"
+    from_port   = 0
+    to_port     = 65535
+    protocol    = "tcp"
+    security_groups = [
+      "sg-0c50cf775611d9db2",
+    ]
+  }
+  egress {
+    description      = "allow all outbound traffic from screener"
```

Comment:
```suggestion
    description      = "allow all outbound traffic from load balancer"
```

Reply:
note: technically I think this only needs egress towards the eligibility screener, unless there are other things it needs to make requests to (like a DNS server or something.) leaving it more open seems okay for now.

Reply:
Personally I'd prefer to narrow the security groups as much as possible, so I'll look more into that